src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        String color = supplier.getRandomColor();
+        int num = new Random().nextInt(100);
```

Comment:
Magic number 100 detected. It's better to declare it as a constant field with a clear name that indicates its purpose.

Reply:
not fixed